### PR TITLE
Fixed typos and spelling in docs from Spellcaster

### PR DESCRIPTION
**Pull request by areibman**
## 📥 Pull Request

**📘 Description**
Updated documentation according to recommendations from Spellcaster https://github.com/AgentOps-AI/Spellcaster

**🎯 Goal**
Cleaner docs are good!

### CODE REVIEW

File: docs/v1/concepts/sessions.mdx [hunk 1]
```diff
@@ -90,8 +90,8 @@ Both processes will now contribute data to the same session.
 ## The AgentOps SDK Client
 _More info for the curious_
 
-Under the hood, `agentops.init()` sets up a `Client` object with various configuration options like your API key, worker thread options
-for when to send out batches of events, etc. Whenever you start a new session, these configuration options will automatically
+Under the hood, `agentops.init()` sets up a `Client` object with various configuration options such as your API key and worker thread options
```
Comment (HowieG) [inline docs/v1/concepts/sessions.mdx]: It does API Key capital K everywhere else but not here

Comment (HowieG) [inline docs/v1/concepts/sessions.mdx]: This doesn't communicate the same message but I can reword

File: docs/v1/integrations/cohere.mdx [hunk 2]
```diff
@@ -54,7 +54,7 @@ This is a living integration. Should you need any added functionality message us
     </span>
 	</Step>
   <Step title="Run your Agent">
-		Execute your program and visit [app.agentops.ai/drilldown](https://app.agentops.ai/drilldown) to observe your Agents! 🕵️
+		Execute your program and visit [app.agentops.ai/drilldown](https://app.agentops.ai/drilldown) to observe your agents! 🕵️
```
Comment (HowieG) [inline docs/v1/integrations/cohere.mdx]: What do we want here

**Review by HowieG — DISMISSED** [review]

**Comment by HowieG** [comment]
Hell yeah this is sick

**Review by siyangqiu — APPROVED** [review]

**Pull request merged**